lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -464,6 +483,18 @@ def get_capabilities(self):
         result['network_api'] = 'nxapi'
         return result
 
+    def read_module_context(self, module_key):
+        Connection('')
```

Comment:
Why is this call to `Connection('')` needed?

Reply:
Good catch.